### DISCUSSION TITLE
Documentation UI/UX: Search and Landing Page

### DISCUSSION THREAD

**Discussion by philleepflorence** [directus/directus#4327] [Feature Requests]
## What

Add search to the Directus Documentation Module.

## Why

Improve Discovery - See [Documentation: A Discovery Summary and Guide.](https://github.com/directus/directus/discussions/4326)  
Search will allow the user to easily locate the appropriate documentation, especially when the menu does not reflect the docs directory tree.

## How

Add a landing page with a search field and introduction to the different sections of [Directus Documentation](https://docs.directus.io/getting-started/introduction/#documentation).
On build, parse all the headings (H1 - H6) in all docs files into a logical list of text:path.  

## Where

Directus Admin Application - Documentation.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
